package.json

```diff
@@ -64,6 +64,7 @@
     "file-saver": "1.3.8",
     "floatthead": "2.1.2",
     "font-awesome": "4.7.0",
+    "fs-extra": "6.0.1",
```

Comment:
Why adding this?

Reply:
I was unable to do an 'npm install', even after removing the node_modules directory. :(